functions/menu.bash

```diff
@@ -47,10 +47,10 @@ show_main_menu() {
     openhabian_update
 
   elif [[ "$choice" == "02"* ]]; then
-    system_upgrade
+    wait ${PID_APT} 2>/dev/null; system_upgrade
```

Comment:
Can we move this into a function with a speaking name, please? It's impossible to maintain this in the long run.

Reply:
ok now?